FirebaseAppDistribution/Sources/FIRAppDistributionUIService.m

```diff
@@ -294,9 +294,17 @@ - (void)screenshotDetected:(NSNotification *)notification {
   dispatch_async(dispatch_get_global_queue(DISPATCH_QUEUE_PRIORITY_HIGH, 0ul), ^{
     [FIRFADInAppFeedback getManuallyCapturedScreenshotWithCompletion:^(UIImage *screenshot) {
       dispatch_async(dispatch_get_main_queue(), ^{
-        [self startFeedbackWithAdditionalFormText:self.additionalFormText
-                                      releaseName:self.releaseName
-                                            image:screenshot];
+        if ([[FIRAppDistribution appDistribution] isTesterSignedIn]) {
+          [self startFeedbackWithAdditionalFormText:self.additionalFormText image:screenshot];
+        } else {
+          [[FIRAppDistribution appDistribution]
+              signInTesterWithCompletion:^(NSError *_Nullable error) {
+                if (error) {
+                  return;
```

Comment:
I see that you also try and identify the release here. Is there a scenario (other than a transient network call) where it would fail initially but succeed here?

Reply:
This only checks whether it's signed in - doesn't check for a release